nmea/component.go

```diff
@@ -6,8 +6,8 @@ import (
 
 	"github.com/golang/geo/r3"
 	geo "github.com/kellydunn/golang-geo"
+	"github.com/viam-modules/gps/gpsutils"
 	"go.viam.com/rdk/components/movementsensor"
-	"go.viam.com/rdk/components/movementsensor/gpsutils"
```

Comment:
@martha-johnston when you removed the package from rdk, you didn't do the corresponding package change to the gpsutils in this repo - this is something to watch out for in the future - as it does mean updating rdk immediately caused a break in this module.